yucca/run/run_inference.py

```diff
@@ -168,6 +159,8 @@ def main():
     save_softmax = args.save_softmax
     use_wandb = not args.no_wandb
 
+    batch_size = 1  # batch size is fixed to 1 for inference, as write_predictions is implemented on_batch_end
```

Comment:
sorry for being unclear, but I think we should remove it from here because it's actually specified in the DataModule and here I think it makes people think they can actually change it. They cannot as it is right now.
This is from the DataModule:

return DataLoader(self.pred_dataset, num_workers=self.num_workers, batch_size=1)